From: Alasdair G Kergon <agk@redhat.com>
Subject: Remove duplicate module name from messages

Remove explicit module name from messages as the macro now includes it
automatically.

Signed-off-by: Alasdair G Kergon <agk@redhat.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>

---
 drivers/md/dm-mpath.c       |    8 +++-----
 drivers/md/dm-raid1.c       |    5 ++---
 drivers/md/dm-round-robin.c |    2 +-
 3 files changed, 6 insertions(+), 9 deletions(-)

Index: linux/drivers/md/dm-mpath.c
===================================================================
--- linux.orig/drivers/md/dm-mpath.c	2007-07-12 17:03:58.000000000 +0100
+++ linux/drivers/md/dm-mpath.c	2007-07-12 17:04:01.000000000 +0100
@@ -1352,15 +1352,14 @@ static int __init dm_multipath_init(void
 
 	r = dm_register_target(&multipath_target);
 	if (r < 0) {
-		DMERR("%s: register failed %d", multipath_target.name, r);
+		DMERR("register failed %d", r);
 		kmem_cache_destroy(_mpio_cache);
 		return -EINVAL;
 	}
 
 	kmultipathd = create_workqueue("kmpathd");
 	if (!kmultipathd) {
-		DMERR("%s: failed to create workqueue kmpathd",
-				multipath_target.name);
+		DMERR("failed to create workqueue kmpathd");
 		dm_unregister_target(&multipath_target);
 		kmem_cache_destroy(_mpio_cache);
 		return -ENOMEM;
@@ -1381,8 +1380,7 @@ static void __exit dm_multipath_exit(voi
 
 	r = dm_unregister_target(&multipath_target);
 	if (r < 0)
-		DMERR("%s: target unregister failed %d",
-		      multipath_target.name, r);
+		DMERR("target unregister failed %d", r);
 	kmem_cache_destroy(_mpio_cache);
 }
 
Index: linux/drivers/md/dm-raid1.c
===================================================================
--- linux.orig/drivers/md/dm-raid1.c	2007-07-12 16:53:59.000000000 +0100
+++ linux/drivers/md/dm-raid1.c	2007-07-12 17:04:01.000000000 +0100
@@ -1335,8 +1335,7 @@ static int __init dm_mirror_init(void)
 
 	r = dm_register_target(&mirror_target);
 	if (r < 0) {
-		DMERR("%s: Failed to register mirror target",
-		      mirror_target.name);
+		DMERR("Failed to register mirror target");
 		dm_dirty_log_exit();
 	}
 
@@ -1349,7 +1348,7 @@ static void __exit dm_mirror_exit(void)
 
 	r = dm_unregister_target(&mirror_target);
 	if (r < 0)
-		DMERR("%s: unregister failed %d", mirror_target.name, r);
+		DMERR("unregister failed %d", r);
 
 	dm_dirty_log_exit();
 }
Index: linux/drivers/md/dm-round-robin.c
===================================================================
--- linux.orig/drivers/md/dm-round-robin.c	2007-07-12 16:53:59.000000000 +0100
+++ linux/drivers/md/dm-round-robin.c	2007-07-12 17:04:01.000000000 +0100
@@ -205,7 +205,7 @@ static void __exit dm_rr_exit(void)
 	int r = dm_unregister_path_selector(&rr_ps);
 
 	if (r < 0)
-		DMERR("round-robin: unregister failed %d", r);
+		DMERR("unregister failed %d", r);
 }
 
 module_init(dm_rr_init);
